pymeasure/adapters/prologix.py

```diff
@@ -148,13 +154,14 @@ def write_binary_values(self, command, values, **kwargs):
         super().write_binary_values(command, values, **kwargs)
         self.connection.write(b'\n')
 
-    def read(self):
+    def _read(self, **kwargs):
         """ Reads the response of the instrument until timeout
 
         :returns: String ASCII response of the instrument
         """
         self.write("++read eoi")
-        return b"\n".join(self.connection.readlines()).decode()
+        self.log.debug(("WRITE:", "++read eoi"))
```

Comment:
calling `self.write` should log this properly, no?

Reply:
Oh, yes